app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -1,37 +1,43 @@
 import * as yup from "yup";
 
-export const NOTES_FORM_INITIAL_FORM_VALUES = {
+export const NOTES_FORM_INITIAL_VALUES = {
   title: "",
   description: "",
-  tagsList: [],
+  assignedContactId: "",
+  tag: "",
 };
 
+export const NOTES_SEED_VALUES = {
+  title: "How to Claim Warranty?",
+  description:
+    "Description Goes Here. Description Goes Here. Description Goes Here. Description Goes Here. Description Goes Here. Description Goes Here. Description Goes Here. Description Goes Here. Description Goes Here. ",
+  assignedContactId: "1",
+  tag: "Getting Started",
+  created_at: "2023-08-29T19:01:08+05:30",
+};
 export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
   title: yup.string().required("Title is required"),
   description: yup.string().required("Description is required"),
-  tagsList: yup
-    .array()
-    .required("Please select a tag")
-    .min(1, "Please select a tag"),
+  // tagsList: yup
+  //   .array()
+  //   .required("Please select a tag")
+  //   .min(1, "Please select a tag"),
```

Comment:
Whatever code is being merged into the main branch should be kept in its best possible state. So please remove these comments since they are not used anywhere.